src/models/Factor.js

```diff
@@ -244,20 +244,21 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
             'delay': PUSH_INTERVAL
           };
           if (trans.status === 'MFA_CHALLENGE' && trans.poll) {
+            // In verify case since we poll we dont return the trans
```

Comment:
nit: just a little bit more explicit.
```
When the factor has a poll link, we don't return the transaction here. 
Rather we set it and start polling.
```

Reply:
I moved the setTranasction outside If and the code now is readable. Just added a small comment now.